pro/src/new_components/StockFormRow/StockFormRow.module.scss

```diff
@@ -0,0 +1,26 @@
+@use 'styles/mixins/_rem.scss' as rem;
+
+.stock-form-row {
+  display: flex;
+  // justify-content: center;
+  align-items: center;
+  padding: 0 rem.torem(24px);
+
+  .stock-form {
+    display: flex;
+    align-items: center;
+    > *:not(:last-child) {
+      margin-right: rem.torem(24px);
+    }
+    margin-right: rem.torem(24px);
```

Comment:
hm y a 2 fois la même chose du coup soit y a une erreur sur une des marges soit le not last child sert aps :p

Reply:
```css
.stock-form {
    margin-right: rem.torem(24px);
```
=> margin sur `stock-form`
```js
.stock-form {
      > *:not(:last-child) {
        margin-right: rem.torem(24px);
      }
```
=> margin sur les enfants:`stock-form > *:not(:last-child)`